app/models/event.rb

```diff
@@ -6,7 +6,7 @@ class Event < ActiveRecord::Base
   validates :end_date, date: { after_or_equal_to: :start_date }
   validates :organizer_email, confirmation: true, format: { with: /.+@.+\..+/ }, presence: true
   validates :organizer_email_confirmation, presence: true, on: :create
-  validates :website, :code_of_conduct, format: { with: /.+\..+/}
+  validates :website, :code_of_conduct, format: { with: /(http|https):\/\/.+\..+/ }
```

Comment:
I wonder if that could be confusing. I'm guessing a lot of people would put something formatted like www.blah.org/coc or even just blah.org/coc, and then the user experience of that not being valid might suck a bit?
I don't think there'd be a real problem with people putting nonsense if they want their event approved, so maybe we don't need a very strong validation for this one?